wgpu-types/src/lib.rs

```diff
@@ -388,6 +388,13 @@ bitflags::bitflags! {
         ///
         /// This is a native-only feature.
         const TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES = 0x0000_0000_1000_0000;
+        /// Enables 64-bit floating point types in SPIR-V shaders.
+        ///
+        /// Supported Platforms:
+        /// - Vulkan
```

Comment:
What is causing this to be limited to only vulkan? Dx12/metal should support it i believe.

Reply:
This is just based on which hal backends expose support for the feature. There might also be limitations from the shader translation, but that I'm less sure of

Reply:
Though looking at the Metal Shading Language Specification:
> Metal does not support the double, long long, unsigned long long, and long double data types